pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -181,7 +181,7 @@ func (alloc *Action) allocateResourcesForTasks(tasks *util.PriorityQueue, job *a
 		klog.V(3).Infof("There are <%d> nodes for Job <%v/%v>", len(ssn.Nodes), job.Namespace, job.Name)
 
 		if err := ssn.PrePredicateFn(task); err != nil {
-			klog.V(3).Infof("PrePredicate for task %s/%s failed for: %v", task.Namespace, task.Name, err)
+			klog.Errorf("PrePredicate for task %s/%s failed for: %v", task.Namespace, task.Name, err)
```

Comment:
can we use level 4？I remember there is a change about the log level from error to info. Please help to check it.

Reply:
ok, I also want to ask, how to distinguish different log levels? I saw that there are levels 3, level 4, and level 5 in the code.

Reply:
done